src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import { getUser, getFolder } from "../api";
+import "./App.css";
+import NavigationBar from "./NavigationBar";
+import { useEffect, useState } from "react";
+import FolderBar from "./FolderBar";
+import CardSection from "./CardSection";
+import Footer from "./Footer";
+
+function App() {
+  const [profile, setProfile] = useState(null);
+  const [folderName, setFolderName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [profileImage, setProfileImage] = useState("");
+  const [cardList, setCardList] = useState(null);
```

Comment:
배열데이터는 기본값을 빈 배열로 초기화해주는게 좋아요~
```suggestion
  const [cardList, setCardList] = useState([]);
```